content/oci/oci_test.go

```diff
@@ -1993,3 +2042,114 @@ func equalDescriptorSet(actual []ocispec.Descriptor, expected []ocispec.Descript
 	}
 	return true
 }
+
+func TestStore_Delete(t *testing.T) {
```

Comment:
Can some/most of this be deduplicated with the memory_test TestStoreDelete with similar contents?